kubectx

```diff
@@ -22,7 +22,10 @@ set -eou pipefail
 IFS=$'\n\t'
 
 SELF_CMD="$0"
-KUBECTX="${XDG_CACHE_HOME:-$HOME/.kube}/kubectx"
```

Comment:
I wonder if you want to move that file to the new path, if it exists, for backward compatibility?

Reply:
Yeah I think we should keep $KUBECTX the same, but use the new $KUBECTX_CACHE_DIR while determining $CONTEXT_LIST_CACHE path.

Besides, `~/.kube/cache` can be nuked occassionally, so it would be good to keep the last-context file outside `cache` dir.

Reply:
I restored it in its original path but renaming it `KUBECTX_PREVIOUS_VALUE`. Do you think we should stick to `KUBECTX` only?